.circleci/config.yml

```diff
@@ -12,10 +14,38 @@ jobs:
       - run: yarn
       - run: yarn lint
       - run: yarn check-circular
+      # Run jest tests
       - run:
+          name: Run tests
           command: yarn test-once
-          environment:
-            NODE_OPTIONS: --max_old_space_size=4096
+          when: always
+      # Run coverage
+      - run:
+          name: Run coverage
+          command: yarn run cover
+          when: always
+      # Run coverage report for Code Climate
+      - run:
+          name: Setup Code Climate test-reporter
+          command: |
+              # download test reporter as a static binary
+              curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+              chmod +x ./cc-test-reporter
+              ./cc-test-reporter before-build
+          when: always
+      - run:
+          name: Send coverage report to Code Climate
+          command:
+              ./cc-test-reporter after-build -t lcov
+          when: always
+      # Upload results
+      - store_artifacts: # upload test coverage as artifact
+          path: ./coverage/lcov.info
+          prefix: tests
+      # - run:
```

Comment:
I see this is not needed anymore
Maybe we'll need the max old space arg so tests don't break randomly?

Reply:
oopsie, I duplicated the line above but forgot to put back the environment